crd/kubeflow.org_mpijobs.yaml

```diff
@@ -7925,9 +7925,6 @@ spec:
                     type:
                       description: Type of job condition.
                       type: string
-                  required:
```

Comment:
these two fields should remain required. A condition without them doesn't make sense.

Reply:
That makes sense.

Reply:
Fixed.